Clinical-Genomics/scout
# Fix #1898 and potential bug

Bug fix #1898

**How to test**:
1. Add analyses and delivery reports from some real case to a test case on stage
1. Check that the link to the delivery reports work

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
